### ISSUE TITLE
Automation Broken with Placeable Bugs

### ISSUE THREAD

**Issue by DrParadox7** [GTNewHorizons/twilightforest#75]
@Gordon-Frohman The refactor you submitted is breaking automation.

From my community:

> Not an incredibly common thing to do with these items(though a neat lighting/alarm alternative), but I am concerned this may branch into other larger problems. I am not sure what other changes were made to the item besides aspects, but the Block Placer places the "out of date" ones just fine, and can't place the new ones.
> Cicada (working, but out of date) item id: 1816
> Cicada (not working, current) item id: 8651:1
> Firefly (working, but out of date) item id: 1815
> Firefly (not working, current) item id: 8657

![2024-06-05_14 22 49](https://github.com/GTNewHorizons/twilightforest/assets/47131096/ff65f9c9-f836-4cd5-85bc-45c91af508e5)
![2024-06-05_14 23 06](https://github.com/GTNewHorizons/twilightforest/assets/47131096/b8f86861-e386-447d-9e90-e0f1307b33bc)
![2024-06-05_14 22 08](https://github.com/GTNewHorizons/twilightforest/assets/47131096/1df3519b-732d-40ea-93be-7caf4c1a7643)

Also are IDs being occupied needlessly with the new refactor?

**Comment by Gordon-Frohman** [comment]
The problem is kind of different
Old critters are blocks, and new ones are items
That's probably the reason why it doesn't work
And I don't know if it can be fixed
What is that automation though?

**Comment by Gordon-Frohman** [comment]
> Also are IDs being occupied needlessly with the new refactor?

Not really. Critters just work like doors/beds/skulls/brewing stands/etc. now - blocks that are placed using an item instead of an itemblock. And because of this you can now wear them on your head
